### PR TITLE
Add support for absolute lookup

### PR DESCRIPTION
**Pull request by tmquinn**

### CODE REVIEW

File: addon/index.js [hunk 1]
```diff
@@ -18,15 +18,32 @@ export default class Resolver {
...
+    const fullNameParts = fullName.split('@');
+
+    // HTMLBars uses helper:@content-helper which collides
+    // with ember-cli namespace detection.
+    // This will be removed in a future release of HTMLBars.
+    if (fullNameParts.length === 2 && fullName !== 'helper:@content-helper') {
+      const prefixParts = fullNameParts[0].split(':');
+
+      if (prefixParts.length === 2) {
+        [type, prefix] = prefixParts;
```
Comment (stefanpenner) [inline addon/index.js]: this wont transpile nicely, as the compiler is unaware if `prefixParts` is an array or an iterable.

File: addon/index.js [hunk 2]
```diff
@@ -18,15 +18,32 @@ export default class Resolver {
...
-    let name = fullName.substring(index + 1, fullName.length);
-    let moduleName = this.namespace.modulePrefix + '/';
+    let prefix, type, name, moduleName;
+
+    const fullNameParts = fullName.split('@');
+
+    // HTMLBars uses helper:@content-helper which collides
+    // with ember-cli namespace detection.
+    // This will be removed in a future release of HTMLBars.
+    if (fullNameParts.length === 2 && fullName !== 'helper:@content-helper') {
```
Comment (stefanpenner) [inline addon/index.js]: maybe the split can be made more intelligent, but hardcoding ^ seems problematic.

I believe ^ relies on the fact that `:@` is not a valid namespace, so our split likely needs to be more aware of this as well.

Comment (tmquinn) [inline addon/index.js]: @rwjblue Is this still a problem in Glimmer?

Comment (rwjblue) [inline addon/index.js]: No, it should not be an issue in Ember 2.10+.

Comment (stefanpenner) [inline addon/index.js]: @tmquinn lets drop it then :) 2.10+ support is fine by me.

**Comment by stefanpenner** [comment]
released as v0.1.0 🎉

**Pull request merged**